src/addons/modal-and-panel/index.scss

```diff
@@ -0,0 +1,2 @@
+@import 'modal';
```

Comment:
@Jeremy-Walton My concern here is the file name being overly specific. Perhaps we can have a naming convention that captures additional controls more generally and not specifically for modals and panels only?

Reply:
A generic `rails-components` might be useful in the future, but I don't anticipate a lot of "rails" specific components being added to the design system. Most of the time our components are very generic and not tied to rails. Modal and panel are because of the JS implementation behind them.

Reply:
I don't feel we have found a lot of component styles with rails specific implementation requirements (JS, etc) that are generic enough to move out of their respective applications and into RoleModel Rails or the RoleModel Design System.

I'm open to the possibility of it happening, but until then, these can stay as `rails-modal-and-panel`